Jan 15, 2024 8:37:59 PM src.io.LoggerDeusto log
SEVERE: No se pudo conectar con la base de datos: path to ' /Users/jonmendizabal/git/Proyecto-Concesionario-de-Coches-Final/ProyectoConcesionario/bin/resources/db/database.sqlite': '/Users/jonmendizabal/git/Proyecto-Concesionario-de-Coches-Final/ProyectoConcesionario/ ' does not exist
